examples/identity/src/main.sw

```diff
@@ -54,7 +54,9 @@ impl IdentityExample for Contract {
     fn access_control_with_identity() {
         // ANCHOR: access_control_with_identity
         let sender = msg_sender().unwrap();
-        require(sender == storage.owner.read(), MyError::UnauthorizedUser(sender));
-        // ANCHOR_END: access_control_with_identity
+        require(sender == storage
+            .owner
+            .read(), MyError::UnauthorizedUser(sender));
```

Comment:
This looks a bit strange, but I'm fine with a follow up if you'd prefer to get this PR merged for now.

Ideally it would look like:
```sway
require(
    sender == storage.owner.read(),
    MyError::UnauthorizedUser(sender),
);
```

Reply:
I like your format better.

Reply:
Does it look better now @eureka-cpu ?